packages/core/src/browser/window/default-secondary-window-service.ts

```diff
@@ -124,6 +135,48 @@ export class DefaultSecondaryWindowService implements SecondaryWindowService {
         return newWindow;
     }
 
+    protected findSecondaryWindowCoordinates(widget: ExtractableWidget): (number | undefined)[] {
+        const clientBounds = widget.node.getBoundingClientRect();
+        const preference = this.preferenceService.get('window.secondaryWindowPlacement');
+
+        let height; let width; let left; let top;
+        const offsetY = 20;
```

Comment:
Could you explain where the 20 is coming from? Maybe as a comment in the code

Reply:
Its an offset to avoid the browser window title bar. I use it to better align the secondary-windows. 
I also added a comment for clarification :+1: